dbms/src/Flash/Coprocessor/RemoteRequest.cpp

```diff
@@ -125,13 +125,37 @@ RemoteRequest RemoteRequest::build(
         tici_scan.constructTiCIScanForRemoteRead(mutable_tici_scan);
 
         const auto & return_columns = tici_scan.getReturnColumns();
```

Comment:
it looks to me that L127 can be moved before L150, limit the scope of `return_columns`